cql/ScreeningLibrary.cql

```diff
@@ -216,6 +216,7 @@ define OrderDate:
 define DecisionAids:
   {
     recommendation: ScreeningRecommendation.short,
+    recommendationId: ScreeningRecommendation.id,
```

Comment:
@mrnosal One question I had where I was hoping to gauge your opinion: should I add IDs to screening library tests? The way the tests are set up, this wouldn’t be required, but would be possible.

Reply:
I don't think this is required for this PR. We can open another ticket to review the screening library tests. Many of them do not include the Recommendation object in the tests anyway, so that would be something to add.

Reply:
I added [CCSMCDS-379](https://jira.mitre.org/browse/CCSMCDS-379).